src/coreclr/src/vm/amd64/asmhelpers.S

```diff
@@ -1,6 +1,6 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
+# Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why the change in comment style?  Makes it difficult to see the actual thing this PR is fixing.

Reply:
The C style comments were messing with my editors code highlighting and since github will highlight the changed part of the line and this change is very small I figured it wouldn't impact readability that much.